Projects/App/Sources/Screens/ClaimsScreen/Screens/HonestyPledge.swift

```diff
@@ -86,6 +86,7 @@ struct DidAcceptPledgeNotifier: View {
                 if value && !hasNotifiedStore {
                     hasNotifiedStore = true
                     store.send(.didAcceptHonestyPledge)
+                    hAnalyticsEvent.honorPledgeConfirmed().send()
```

Comment:
I was wondering if we can start sending the `hAnalyticsEvents` from the Store inside `effects` instead. 

This keeps all the views from having to import `hAnalytics`. I also feel it is more appropriate to place it in `effects` because it is a side effect happening after an action .🤔